strawberry/contrib/django/tests/test_views.py

```diff
@@ -0,0 +1,41 @@
+import json
+
+from django.test.client import RequestFactory
+
+import strawberry
+from strawberry.contrib.django.views import GraphQLView
+
+
+@strawberry.type
+class Query:
+    hello: str = "strawberry"
+
+
+schema = strawberry.Schema(query=Query)
+
+
+def test_playground_view():
+    factory = RequestFactory()
+
+    request = factory.get("/graphql/", HTTP_ACCEPT="text/html")
```

Comment:
can we use django's reverse here?

Reply:
We could, but since we're passing the request object directly to the view instead of performing a request using the test client it doesn't really matter. The path could be any arbitrary string and the behavior of the test will be the same.